create_tailscale_sysext.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+export ARCH="${ARCH-x86-64}"
+SCRIPTFOLDER="$(dirname "$(readlink -f "$0")")"
+
+if [ $# -lt 2 ] || [ "$1" = "-h" ] || [ "$1" = "--help" ]; then
+  echo "Usage: $0 VERSION SYSEXTNAME"
+  echo "The script will download tailscale binaries and create a sysext squashfs image with the name SYSEXTNAME.raw in the current folder."
```

Comment:
```suggestion
  echo "The script will download tailscale binaries (e.g., for 1.64.0) and create a sysext squashfs image with the name SYSEXTNAME.raw in the current folder."
```
We include an example version number to make the format clear (`v` prefix or not) and make testing easy without searching for a version to use.